graphdatascience/call_parameters.py

```diff
@@ -1,9 +1,40 @@
 from collections import OrderedDict
-from typing import Any
+from typing import Any, Optional
+from uuid import uuid4
 
 
 class CallParameters(OrderedDict[str, Any]):
     # since Python 3.6 also initializing through CallParameters(**kwargs) is order preserving
 
     def placeholder_str(self) -> str:
         return ", ".join([f"${k}" for k in self.keys()])
+
+    def get_job_id(self) -> Optional[str]:
+        config = self["config"] if "config" in self else {}
+
+        job_id = None
+        if "jobId" in config:
+            job_id = config["jobId"]
+
+        if "job_id" in config:
+            job_id = config["job_id"]
```

Comment:
duplicated statement
```suggestion
        if "jobId" in config:
            job_id = config["jobId"]
```

Reply:
wait ... this is not quite duplicate. It caters for two different syntaxes `jobId` and `job_id`